ext/ext_skel.php

```diff
@@ -226,6 +226,13 @@ function process_args($argv, $argc) {
 		error('The skeleton directory was not found');
 	}
 
+	// Validate extension name
+	if (!preg_match('/^[a-z][a-z0-9_]+$/i', $options['ext'])) {
+		error('Invalid extension name. Name must start with a letter and'
```

Comment:
Could we reword this a bit to make it more straightforward? Following the description used for PHP variable names in the PHP manual (*"A valid variable name starts with a letter or underscore, followed by any number of letters, numbers, or underscores."*) we could reword it as follows:

```
error('Invalid extension name. Valid names start with a letter,'
    .'followed by any number of letters, numbers, or underscores.'
    .'Using only lower case letters is preferred.');
```